superset/sql_lab.py

```diff
@@ -224,6 +234,38 @@ def execute_sql_statement(sql_statement, query, user_name, session, cursor):
     return dataframe.SupersetDataFrame(data, cursor_description, db_engine_spec)
 
 
+def _serialize_payload(payload, use_msgpack=False):
+    logging.debug("Serializing to msgpack: {}".format(use_msgpack))
```

Comment:
f-strings are the new hotness since py3.6, `.format()` is very last season :)
`f"Serializing to msgpack: {use_msgpack}"`